lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -120,5 +140,69 @@ def display_logs
 
       @already_displayed = true
     end
+
+    class Container
```

Comment:
I consider it very internal. I'm going to try to get coverage of the case Ian mentioned somehow though, and it can't really be an integration test since the intended behaviour is a potentially long timeout. I'll see what I can do.